src/components/Link.stories.tsx

```diff
@@ -19,7 +20,7 @@ export default {
   component: Link,
 };
 
-export const Basic = (args) => <Link {...args} />;
+export const Basic = (args: LinkProps) => <Link {...args} />;
```

Comment:
please prefer using the FunctionComponent<Props>

```suggestion
export const Basic: FunctionComponent<LinkProps> = (props) => <Link {...props} />;
```